packages/@aws-cdk/aws-apigatewayv2/README.md

```diff
@@ -5,15 +5,9 @@
 
 Features                                   | Stability
 -------------------------------------------|--------------------------------------------------------
-CFN Resources                              | ![Stable](https://img.shields.io/badge/stable-success.svg?style=for-the-badge)
```

Comment:
This README file is the odd one out. Rearranging it so that the the script can use the same logic to process all files.